roles/web/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+# Bring up apache2 webserver with PHP7, node and composer
+# Examples:
+#   https://github.com/ansible/ansible-examples/tree/master/lamp_simple
+#   https://github.com/ansible/ansible-examples/blob/master/wordpress-nginx/roles/php-fpm/tasks/main.yml
+
+- name: Install apache2
+  apt: name=apache2 state=latest
+
+- name: Create /var/www
+  command: creates=/var/www/ mkdir /var/www/
+
+- name: enable apache mod rewrite
+  apache2_module: name=rewrite state=present
+  notify:
+    - restart apache
+
+- name: allow apache override to all
+  command: sed -i "s/AllowOverride None/AllowOverride All/g" /etc/apache2/apache2.conf
+  notify:
+    - restart apache
+
+- name: Configure Apache to serve `./public` folder
+  shell: |
+    ln -fs /var/www/configs/apache-hitchwiki.conf hitchwiki.conf chdir=/etc/apache2/sites-available removes=hitchwiki.conf
+    ln -fs ../sites-available/hitchwiki.conf hitchwiki.conf chdir=/etc/apache2/sites-enabled
+    rm 000-default.conf chdir=/etc/apache2/sites-enabled
+    rm -fr "{{ ROOTDIR }}/html" #chdir=/etc/apache2/sites-enabled
+  notify:
+    - restart apache
+
+- name: install PHP7 and extensions
+  apt: name={{item}} state=present
+  with_items: { php7.0, libapache2-mod-php7.0, php7.0-mysql, php7.0-curl, php7.0-gd, php7.0-intl, php7.0-imap, php7.0-mcrypt, php7.0-pspell, php7.0-recode, php7.0-sqlite3, php7.0-tidy, php7.0-xmlrpc, php7.0-xsl, php7.0-mbstring, php7.0-opcache, php-memcache, php-pear, php-imagick, php-apcu, php-gettext }
+
+- name: Turn on PHP errors
+  shell: |
+    sed -i "s/error_reporting = .*/error_reporting = E_ALL/" /etc/php/7.0/apache2/php.ini
+    sed -i "s/display_errors = .*/display_errors = On/" /etc/php/7.0/apache2/php.ini
+  notify:
+    - restart apache
```

Comment:
This should be off by default (at least in prod) and we have [a flat at config](https://github.com/Hitchwiki/hitchwiki/blob/master/configs/mediawiki.php#L59-L63) we use to control this.

Reply:
moved to dev as well now

Reply:
It should still be off by default, so that setting `$hwDebug=false` will hide them.